pkg/recovery/recovery.go

```diff
@@ -45,12 +46,17 @@ type (
 )
 
 // CrashLog write down the current memory and stack info and the cpu/mem/disk infos into log dir
-func CrashLog(r interface{}, cfgLog log.GlobalConfig) {
+func CrashLog(r interface{}, dir string) {
```

Comment:
a fixed string as input is not configurable, `config.System.SystemLogDBPath` is defined but not used, you can use it as the path for crash log file.